components/raftstore/src/store/config.rs

```diff
@@ -299,6 +299,15 @@ pub struct Config {
     #[doc(hidden)]
     pub long_uncommitted_base_threshold: ReadableDuration,
 
+    /// Interval to check pre become leader.
+    #[doc(hidden)]
+    pub pre_become_leader_state_tick_interval: ReadableDuration,
+    #[doc(hidden)]
+    pub exit_pre_become_leader_state_ticks: usize,
+    /// Max tick count for the entry cache to be warmed up
+    /// before a follower becomes leader. Set it to 0 to disable warmup.
+    pub warm_up_raft_entry_cache_ticks: usize,
```

Comment:
what's the difference between exit_pre_become_leader_state_ticks and warm_up_raft_entry_cache_ticks?

Reply:
1. `warm_up_raft_entry_cache_ticks` refers to the timeout of the log fetching operation.
2. `exit_pre_become_leader_state_ticks` refers to the timeout of the leadership transfer operation. When entries are fetched into cache, we should ensure they are not compacted before it becomes leader.

Reply:
1. what's the unit of `warm_up_raft_entry_cache_ticks` and `exit_pre_become_leader_state_ticks`, counts or time? 
2. Could you add the comments for `exit_pre_become_leader_state_ticks`?

Reply:
I'm working on code refactoring to simplify these configurations.

Reply:
Currently, there is only one configuration item.